app/views/admin/administrators/administrators/index.html.erb

```diff
@@ -1,5 +1,5 @@
 <h1 class="govuk-heading-l">Administrators</h1>
-<%= govuk_link_to "Create a new administrator", new_admin_administrator_path, button: true %>
+<%= govuk_link_to "Create a new administrator", new_admin_administrator_path, button: true, "data-test" => "create-admin-button" %>
```

Comment:
Nice - there is however a problem with submit button just to be aware - a workaround is to simply add a class when working with submit buttons

Reply:
Sending a PR fixing that today! :)